desktop/apps/consignments2/index.js

```diff
@@ -21,6 +21,12 @@ app.get(landing.paths.show, adminOnly, routes.landingPage)
 app.get(landing.paths.show + '/data', adminOnly, data)
 app.get(landing.paths.edit, adminOnly, edit)
 app.post(landing.paths.edit, adminOnly, upload)
-app.get('/consign2/submission', adminOnly, routes.submissionFlow)
-app.get('/consign2/submission/:id/thank_you', adminOnly, routes.submissionThankYou)
-app.get('/consign2/submission/:id/upload', adminOnly, routes.submissionUpload)
+app.get('/consign2/submission/:id/describe-your-work', routes.submissionFlowWithFetch)
+app.get('/consign2/submission/:id/upload-photos', routes.submissionFlowWithFetch)
+app.get('/consign2/submission/:id/thank-you', adminOnly, routes.submissionFlowWithId)
+app.get('/consign2/submission/:id/upload', adminOnly, routes.submissionFlowWithId)
+app.get('/consign2/submission', routes.submissionFlow)
+app.get('/consign2/submission/create-account', routes.redirectToSubmissionFlow)
```

Comment:
Yeah, I wasn't sure if that was anti-pattern (like should we be explicitly throwing a 404 if someone tries to visit /consign/submission/foo ?). But definitely cleaner to do it that way.